app/assets/stylesheets/forever_style_guide/modules/_hero_simple.scss

```diff
@@ -0,0 +1,89 @@
+.hero_simple {
+  position: relative;
+  height: 500px;
+  background-size: cover;
+
+  h1, p {
+    color: color('white');
+    text-shadow: color('gray-600') 0 1px 3px;
+  }
+
+  @media (min-width: $screen-lg) {
+    height: 600px;
+  }
+  @media (min-width: $screen-xl) {
+    height: 650px;
+  }
+  @media (min-width: $screen-xxl) {
+    height: 900px;
+  }
+
+  &.hero_simple-sm {
+    height: auto;
+    @media (max-width: $screen-xs) {
+      padding-top: 20px;
+      padding-bottom: 20px;
+    }
+    @media (min-width: $screen-xs) {
+      height: 230px;
+    }
+    @media (min-width: $screen-md) {
+      height: 260px;
+    }
+  }
+
+  &.hero_simple-sm-action {
+    // adding an additional element makes vertical spacing too tricky to be worth messing with a fixed height here 
+    @media (min-width: $screen-xs) and (max-width: $screen-sm){
+      height: auto;
+      padding-top: 30px;
+      padding-bottom: 30px;
+    }
+  }
+}
+.hero_simple-block {
+  position: absolute;
+  bottom: 0;
+  padding: 30px;
+
+  @media (min-width: $screen-xxl) {
+    padding: 50px;
+  }
+}
+.hero_simple-block-cta {
+  margin-top: 5px;
+
+  // for hero CTAs layout in between xs -> sm breakpoints
+  @media (max-width: 580px) {
```

Comment:
This should be a named variable rather than a magic number. More broadly, there's a lot of magic numbers in this doc, which is the way the standard hero module is handled currently. But it could be worth considering replacing a lot of these w/ variables for improved readability and repurposeability.